.github/scripts/shellCheck.sh

```diff
@@ -1,48 +1,39 @@
 #!/bin/bash
 
-CURRENT_DIR=$(pwd)
-ROOT_DIR="$(dirname "$(dirname "$(dirname "$(realpath "${BASH_SOURCE[0]}")")")")"
+ROOT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")/../.." &>/dev/null && pwd)
+readonly ROOT_DIR
 
-cd "$ROOT_DIR" || exit 1
+source "$ROOT_DIR"/.github/scripts/shellUtils.sh
 
-source ./.github/scripts/shellUtils.sh
-
-declare -r DIRECTORIES_TO_IGNORE=(
-  './node_modules'
-  './vendor'
-  './ios/Pods'
-  './.husky'
-)
+readonly DIRECTORIES_TO_IGNORE="\
+-path $ROOT_DIR/node_modules \
+-o -path $ROOT_DIR/vendor \
+-o -path $ROOT_DIR/ios/Pods \
+-o -path $ROOT_DIR/.husky"
```

Comment:
You don't need the line continuations, IMO it's a little more readable without them

```suggestion
readonly DIRECTORIES_TO_IGNORE="
    -path $ROOT_DIR/node_modules
    -o -path $ROOT_DIR/vendor
    -o -path $ROOT_DIR/ios/Pods
    -o -path $ROOT_DIR/.husky
"
```